martian/src/metadata.rs

```diff
@@ -271,40 +260,13 @@ impl Metadata {
         Error::new(e).context(context)
     }
 
-    fn _append(&mut self, name: &str, message: &str) -> Result<()> {
-        let filename = self.make_path(name);
-        let mut file = OpenOptions::new()
-            .create(true)
-            .append(true)
-            .open(filename)?;
-        file.write_all(message.as_bytes())?;
-        file.write_all(b"\n")?;
-        // Ensure the file is closed before we write the journal, to reduce
-        // the chances that `mrp` sees the journal entry before the file content
-        // has be sync'ed.  This can be an issue on nfs systems.
-        drop(file);
-        self.update_journal(name)?;
-        Ok(())
+    pub(crate) fn alarm_file(&self) -> &SharedFile {
+        &self.alarm_file
     }
 
-    /// Write to _log
-    pub fn log(&mut self, level: &str, message: &str) -> Result<()> {
```

Comment:
For those not in our slack conversation, turns out this is unnecessary because we're already overriding the global-scope logging macros to write to the log.  And we weren't using it.  And we shouldn't, because this was buggy, as
1. It didn't use `write_all`.
2. if it panicked or if `write` or `flush` returned an error, it would close the file descriptor.  Which means that if you continued on anyway and opened some other file, it would get fd 3, and then the next time you tried logging you'd log into whatever that file was.